public-event-list.md

```diff
@@ -136,268 +138,566 @@ To get the custom included in the Public Event List payload, first create a Fiel
 ### Example
 ```javascript
 {
-  "venues": {
-    "a05800000070JMaAAM": {
-      "id": "a05800000070JMaAAM",
-      "name": "Falcon Theatre",
-      "lastModifiedDate": "2012-08-06T16:28:39.000Z",
-      "detail": "<p>4252 Riverside Dr. <br />Burbank, CA 91505</p>\n<p>(818) 955-8101 <br /><br />Corner of Rose St. and Riverside Dr., across from the historic Bob's Big Boy restaurant in Toluca Lake. <br /><br />Free parking available.<br /><br />All sales final, no refunds, no exchanges.</p>",
-      "address": null
+  "venues" : {
+    "a1A8A000001ZPFUUA4" : {
+      "name" : "Allentown Symphony Hall",
+      "lastModifiedDate" : "2021-03-26T16:55:08.000Z",
+      "id" : "a1A8A000001ZPFUUA4",
+      "detail" : null,
```

Comment:
Let's add sample `detail` and `address` to show that it can contain HTML (can address contain HTML?)